lib/util/http/jira.spec.ts

```diff
@@ -0,0 +1,33 @@
+import * as httpMock from '../../../test/http-mock';
+import * as hostRules from '../host-rules';
+import { JiraHttp, setBaseUrl } from './jira';
+
+describe('util/http/jira', () => {
+  const api = new JiraHttp();
+
+  it('throws error if setBaseUrl not called', async () => {
+    await expect(api.postJson('some-path')).rejects.toThrow(
+      TypeError('Invalid URL')
+    );
+  });
+
+  it('accepts custom baseUrl', async () => {
+    const siteUrl = 'https://some-site.atlassian.com';
+    httpMock.scope(siteUrl).post('/some-path').reply(200, {});
```

Comment:
you should add required header to validate that correct auth is send, otherwise you can fully remove auth, as it's already tested.

Reply:
ok, removed